app/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,5 @@
 <resources>
     <string name="app_name">MinuteJournal</string>
-<<<<<<< HEAD
```

Comment:
This is usually the result of an unmerged conflict, and should be removed (as the reviewer has done here).